sdk/communication/azure-communication-phonenumbers/swagger/README.md

```diff
@@ -95,3 +95,39 @@ directive:
       $["properties"]["assignmentType"].readOnly = true;
       $["properties"]["capabilities"].readOnly = true;
 ```
+
+### Rename PhoneNumberCapabilities Calling PhoneNumberCapabilityValue to PhoneNumberCapabilityType
```

Comment:
Should we do this in all languages, or maybe the swagger?

Reply:
Sure, I think it might be easier to update the swagger if we plan to do this in all languages. @DominikMe what do you think?